marmoset/pxe/client_config.py

```diff
@@ -156,14 +159,15 @@ def __write_config_file(self, content, path=None):
         if path is None:
             path = self.file_path()
 
+        # pylint seems to be wrong on this one
+        #pylint: disable-msg=unexpected-keyword-arg
         os.makedirs(ClientConfig.CFG_DIR, exist_ok=True)
-        f = open(path, 'w')
-        f.write(content)
-        f.close()
+        file = open(path, 'w')
+        file.write(content)
+        file.close()
```

Comment:
use the "with open" function here, is it good practice according to the python documentation:
https://docs.python.org/3.5/tutorial/inputoutput.html#methods-of-file-objects (last example)